package.json

```diff
@@ -56,5 +56,16 @@
     "hooks": {
       "pre-commit": "lerna run --stream --parallel precommit"
     }
+  },
+  "resolutions": {
```

Comment:
yesterday in this PR https://github.com/BitGo/BitGoJS/commit/afbe4d6da5a8c799f2fcc49059054361847a273b#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L60 resolutions was deleted from master, maybe this is no longer necessary